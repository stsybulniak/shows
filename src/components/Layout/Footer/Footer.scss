.Footer {
  background-color: var(--color-primary);
  height: 10rem;
  color: white;
  display: flex;
  align-items: center;
  justify-content: center;
  font-size: 3rem;
}