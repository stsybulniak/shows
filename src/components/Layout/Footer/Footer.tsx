import { FC } from 'react';
import './Footer.scss';

const Footer: FC = () => <footer className='Footer'>Footer</footer>;

export default Footer;
