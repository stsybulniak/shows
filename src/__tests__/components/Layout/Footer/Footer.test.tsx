import { render } from '@testing-library/react';
import Footer from '../../../../components/Layout/Footer/Footer';

describe('<Footer />', () => {
  it('Footer matches snapshot', () => {
    const { baseElement } = render(<Footer />);
    expect(baseElement).toMatchSnapshot();
  });
})
