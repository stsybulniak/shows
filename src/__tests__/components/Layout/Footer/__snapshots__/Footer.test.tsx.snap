// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Footer /> Footer matches snapshot 1`] = `
<body>
  <div>
    <footer
      class="Footer"
    >
      Footer
    </footer>
  </div>
</body>
`;
